jax_tqdm/pbar.py

```diff
@@ -93,3 +93,85 @@ def wrapper_progress_bar(carry, x):
         return wrapper_progress_bar
 
     return _progress_bar_scan
+
+
+def progress_bar_fori_loop(
+    n: int, message: typing.Optional[str] = None
+) -> typing.Callable:
+    """
+    Progress bar for a JAX fori_loop
+    Parameters
```

Comment:
Should have an empty line above

Reply:
you mean 1 empty line above `"Progress bar for a JAX fori_loop"`

Reply:
Ah or rather above "parameters"

Reply:
Above "parameters"